index.js

```diff
@@ -192,8 +192,12 @@ var Module = (function () {
 
         if (this.definitions[name] && typeof this.definitions[name] === 'function') {
           return instantiate.apply(this, [this.definitions[name]].concat(Array.prototype.slice.call(arguments, 1)));
-        } else {
-          throw new Error(name + ' is not defined');
+        } else if (typeof require !== 'undefined') {
+          try {
+            return instantiate.apply(this, [define(this, DSController, name, require(name))].concat(Array.prototype.slice.call(arguments, 1)));
+          } catch (e) {
+            throw new Error(name + ' is not defined');
+          }
```

Comment:
I think it's just hiding browserify's error.

The Error is a programmer error, for sure, if they're trying to require something that doesn't exist, and the browserify error will be more descriptive than ours, right?  You changed a if/else into a if/elseif.  It should really be a if/elseif/else for when there is no browserify and we still want the error.